### PR TITLE
DAH 1336 additional housing opportunities page

### PR DESCRIPTION
**Pull request by cade-exygy**
[DAH-1336](https://sfgovdt.jira.com/browse/DAH-1336)

Changes:
 - Added additional housing opportunity resources
 - Pull in updated InfoCardGrid from Bloom UIC
 - Added resource copy to en.json
 - Override css variables
 - Add e2e tests
 -FIx some contrast errors based on WAVE analysis

Review Steps:
- Go to /additional-resources?react=true
- Ensure everything looks okay, is responsive, accessible, etc.
- Compare to Figma (linked in ticket)

[DAH-1336]: https://sfgovdt.jira.com/browse/DAH-1336?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by cade-exygy** [comment]
Waiting on React 18 upgrade

**Review by chadbrokaw-exygy — APPROVED** [review]
Talked offline about some styling things, otherwise looks great!

File: app/assets/json/additional-resources.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    "categories": [
+        {
+            "title": "assistance.additionalResources.sfHousingResources.title",
+            "subtitle": "assistance.additionalResources.sfHousingResources.subtitle",
+            "resources": [
+                {
+                    "title": "assistance.additionalResources.sfHousingResources.fcfs.title",
+                    "agency": "assistance.additionalResources.sfHousingResources.fcfs.agency",
+                    "description": "assistance.additionalResources.sfHousingResources.fcfs.description",
+                    "externalUrl": "https://sf.gov/reports/january-2023/first-come-first-served-bmr-listings"
```
Comment (jimlin-sfgov) [inline app/assets/json/additional-resources.json:11]: This URL and the next two are out-of-date, but the website redirects to the most updated pages, so it should be fine.

**Review by jimlin-sfgov — APPROVED** [review]
Looks good! Would love to get this merged in as soon as possible.

**Pull request merged**